internal/debug/flags.go

```diff
@@ -341,8 +331,8 @@ func StartPProf(address string, withMetrics bool) {
 func Exit() {
 	Handler.StopCPUProfile()
 	Handler.StopGoTrace()
-	if closer, ok := logOutputStream.(io.Closer); ok {
-		closer.Close()
+	if logOutputF != nil {
+		logOutputF.Close()
```

Comment:
As far as I can tell, nothing ever sets the `logOutputIF` ?

Reply:
Hm, the same problem can be said for current `master` too, ahem..

Reply:
Perhaps you shound make the `logOuputIF` into an `io.Closer` instead

Reply:
Can we just remove `logOutputF` and not explicitly close the log file on shutdown?

Reply:
Hm.. I think it's prudent to close files if we have them open. It's not that big of a hassle